docs/General/getting-started/getting-started-with-bpmn-js.md

```diff
@@ -10,3 +10,242 @@ grand_parent: "General"
 ---
 
 # Getting Started With bpmn-js
+
+bpmn-js is a [BPMN 2.0](https://www.omg.org/spec/BPMN/2.0/About-BPMN/) rendering toolkit and web modeler. It allows to view and edit BPMN 2.0 diagrams in the browser without any backend requirement.
+
+Feel free to play with it [here](https://demo.bpmn.io/s/start).
+
+![image](/assets/images/bpmn-start.gif)
+
+There are two ways to include bpmn-js in your application:
+
+* An [all-in-one pre-packaged version of the library](https://unpkg.com/browse/bpmn-js/dist/) allowing you to quickly add BPMN to any web application.
+* The [npm](http://npmjs.com/package/bpmn-js) version, which is more complicated to set up but gives you access to individual library components and allows for easier extensibility.
+
+## Including the Root Element
+
+Both as a viewer or modeler, bpmn-js requires a root DOM element (a simple DIV) on which the rendering is performed. Simply, create a DIV element in the HTML file:
+
+```html
+<!-- BPMN diagram container -->
+<div id="canvas"></div>
+```
+
+## Including the Stylesheet
+
+We should include [diagram-js](/general/getting-started/diagram-js) CSS and [BPMN Icon Font](https://github.com/bpmn-io/bpmn-font) in order to ensure that everything looks good. There're several ways we can achieve that:
```

Comment:
We should write in the same form. So before we wrote `Create a DIV element in the HTML file`.

Therefore also here I would write second person imperative `Include [diagram-js] CSS and ...` and omitt the `should` and `we`

Reply:
It's a valid point that we should follow the same format throughout the article. I updated other phrases to go well with the **we format**. Personally I'd like to keep the **we format** since I find it makes it more engaging to follow to `getting started` articles. I like the current tone of the article.

Reply:
This is the documentation, not a tutorial. Therefore, I'd ditch the _we_.

Reply:
How does Zeebe and Camunda deal with we vs. you? I'd argue we should simply adopt the existing tone for the docs.